### PR TITLE
Add maps chapter

### PR DESCRIPTION
**Pull request by LorenzBischof**

### CODE REVIEW

File: content/en/docs/overview/maps.md [hunk 1]
```diff
@@ -1,9 +1,78 @@
...
+
+The `make` function returns a map of the given type, initialized and ready for use.
+
+{{<go-playground>}}
+package main
+
+import "fmt"
+
+func main() {
+    m := make(map[string]int)
```
Comment (jyniybinc) [inline content/en/docs/overview/maps.md:20]: Remove `make`

File: content/en/docs/overview/maps.md [hunk 2]
```diff
@@ -1,9 +1,78 @@
...
+source:
+- https://go.dev/blog/maps
 ---
 
-## Maps
+A map maps keys to values.
 
-```
-```
+The zero value of a map is `nil`. A `nil` map has no keys, nor can keys be added.
```
Comment (jyniybinc) [inline content/en/docs/overview/maps.md:10]: To initialize a map we use the map literal

File: content/en/docs/overview/maps.md [hunk 3]
```diff
@@ -1,9 +1,78 @@
...
+
+{{<go-playground>}}
+package main
+
+import "fmt"
+
+func main() {
+    m := make(map[string]int)
+    m["route"] = 66
+    i, ok := m["route"] // try changing this value
```
Comment (jyniybinc) [inline content/en/docs/overview/maps.md:56]: Add `if exists`

File: content/en/docs/overview/maps.md [hunk 4]
```diff
@@ -1,9 +1,78 @@
...
+func main() {
+    m := map[string]int{
+        "route": 66,
+    }
+    m["another"] = 50
+    fmt.Println(m)
+}
+{{</go-playground>}}
+
+We can check if a key exists:
```
Comment (jyniybinc) [inline content/en/docs/overview/maps.md:46]: More description

**Review by dvob — APPROVED** [review]

**Pull request merged**